packages/squiggle-lang/src/rescript/Reducer/Reducer_MathJs/Reducer_MathJs_ToCodeTree.res

```diff
@@ -0,0 +1,111 @@
+module CTT = Reducer_CodeTree_T
+module CTV = Reducer_Extension.CodeTreeValue
+module JsG = Reducer_Js_Gate
+module MJ = Reducer_MathJs_Parse
+module Rerr = Reducer_Error
+module Result = Belt.Result
+
+type codeTree = CTT.codeTree
+type codeTreeValue = CTV.codeTreeValue
+type reducerError = Rerr.reducerError
+
+// TODO:
```

Comment:
are these TODOs current and in the right spot?

Reply:
I am removing those redundant todo's because if support a new mathjs node then the code does not compile anyway...

Reply:
ok, gonna resolve this.